defaults/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+fail2ban_bantime: 3600
+fail2ban_findtime: 600
+fail2ban_maxretry: 3
+fail2ban_enabled: "false"
+fail2ban_backend: systemd
+fail2ban_logencoding: auto
+fail2ban_ignoreip:
+  - "127.0.0.1/8"
+fail2ban_additional_filters: []
+fail2ban_jails:
```

Comment:
If every entry has a name, there are no duplicates and there needs to be no way of unsetting an entry within our config repo, you could use a dict for this.

Reply:
What's the advantage of using a dict? Is this how it's done in other roles?

Reply:
The advantage is: With a dict you can't accidentally assign a name twice.

Reply:
Also, you could even define a default configuration for a jail in a group yml and override only some attribute of it in a host yml.

Reply:
Do you mean for the `fail2ban_additional_filters` field?

Reply:
I don't see `fail2ban_additional_filters`, I mean `fail2ban_jails`, i.e. the type of `fail2ban_jails` would be a dict with entries such as:
```yml
fail2ban_jails:
  ssh-ddos:
    ports:
      - ssh
    logpath: %(sshd_log)s
    backend: systemd
    enabled: True
``